rpcs3/Emu/CPU/CPUThread.cpp

```diff
@@ -809,6 +809,28 @@ std::string cpu_thread::get_name() const
 	}
 }
 
+u32 cpu_thread::get_pc(const cpu_thread* cpu)
```

Comment:
Why do we need generic "get_pc" function in emucore, not in GUI?

Reply:
Just to think about: emucore headers compile who knows how many times because they are included everywhere because no one cares. Debugger, however, will never get accidentally included.

Reply:
Ah yes, I meant templates you added in the header, not the function itself.

Reply:
Removed those functions.